CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-<!-- ## Unreleased -->
+## Unreleased
+* [breaking] JavascriptDocument#ast is now a `babel.File` rather than a
```

Comment:
below we yell `BREAKING`

Reply:
I was deliberately quieter here because this is not going to break anyone :)